scripts/createPage/builder.js

```diff
@@ -0,0 +1,17 @@
+const upperCaseFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
Instead of redefining this function, go ahead and import it from `common/utils/string-utils`